validator/rpc/wallet_test.go

```diff
@@ -134,11 +133,30 @@ func TestServer_RecoverWallet_Derived(t *testing.T) {
 	}
 	_, err = s.CreateWallet(ctx, reqCreate)
 	require.ErrorContains(t, "create wallet not supported through web", err, "Create wallet for DERIVED or REMOTE types not supported through web, either import keystore or recover")
+	
+	// This defer will be the last to execute in this func. 
+	resetCfgFalse := featureconfig.InitWithReset(&featureconfig.Flags{
+		WriteWalletPasswordOnWebOnboarding: false,
+	})
+	defer resetCfgFalse()
 
+	resetCfgTrue := featureconfig.InitWithReset(&featureconfig.Flags{
+		WriteWalletPasswordOnWebOnboarding: true,
+	})
+	resetCfgTrue()
+	
```

Comment:
Need to figure out this test; basically how to set the writing and have Recover/Create write the password.